go/vt/vtctl/vdiff2.go

```diff
@@ -225,18 +226,18 @@ func commandVDiff2(ctx context.Context, wr *wrangler.Wrangler, subFlags *pflag.F
 type vdiffTableSummary struct {
 	TableName       string
 	State           vdiff.VDiffState
-	RowsCompared    int64
-	MatchingRows    int64
-	MismatchedRows  int64
-	ExtraRowsSource int64
-	ExtraRowsTarget int64
+	RowsCompared    uint64
+	MatchingRows    uint64
+	MismatchedRows  uint64
+	ExtraRowsSource uint64
+	ExtraRowsTarget uint64
```

Comment:
The table uses bigint signed, which is int64: https://github.com/vitessio/vitess/blob/main/go/vt/sidecardb/schema/vdiff/vdiff_table.sql

So now we can hold larger values in memory than we can store on disk. Why? Please note that there's a historical aversion to using unsigned ints in Vitess (I don't personally agree but that's a different matter). That's why e.g. even for things like sequence tables we do not use bigint unsigned columns. 

I would recommend that you keep the changes minimal to those required for the feature want to add.